watchOS/WelcomePage.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+import FudanUI
+import FudanKit
+import ViewUtils
+
+struct WatchWelcomePage: View {
+    @ObservedObject private var campusModel = CampusModel.shared
+    @ObservedObject private var syncronizer = CredentialSynchronizer.shared
+    @Binding private var showLoginSheet: Bool
+    private let syncOperation: () async throws -> Void
+    
+    init(showLoginSheet: Binding<Bool>, syncOperation: @escaping () async throws -> Void) {
+        self._showLoginSheet = showLoginSheet
+        self.syncOperation = syncOperation
+    }
+    
+    var body: some View {
+        ScrollView {
+            VStack {
+                Image(systemName: "person.badge.shield.exclamationmark")
+                    .font(.largeTitle)
+                Text("Not Logged in")
+                    .font(.caption)
+                    .foregroundStyle(.secondary)
+            }
+            .padding(.bottom, 10)
+            
+            AsyncButton {
+                try await syncOperation()
```

Comment:
Also, I think the button should be disabled with a `ProgressView` displayed when the operation is in progress.